frontend/catalyst/jax_primitives.py

```diff
@@ -83,7 +83,7 @@
 from catalyst.utils.extra_bindings import FromElementsOp, TensorExtractOp
 from catalyst.utils.types import convert_shaped_arrays_to_tensors
 
-# pylint: disable=unused-argument,abstract-method,too-many-lines,too-many-statements
+# pylint: disable=unused-argument,too-many-lines,too-many-statements
```

Comment:
Let's leave existing silencing in place until we can guarantee it won't cause failures:
```suggestion
# pylint: disable=unused-argument,abstract-method,too-many-lines,too-many-statements
```